demos/universal-sentence-encoder/README.md

```diff
@@ -20,27 +20,30 @@ universal-sentence-encoder-multilingual/
         └── variables.index
 
 ```
-Temporarily OVMS requires the model in a frozen graph format so a similar structure is to be created.
-The graph format represents the input and output as a tensorname so we will apply a `mapping_tensor.json` to adjust the model server names.
+
+In case the input and output names are changed during the model load in OpenVINO backend, apply a `mapping_config.json` to adjust them from the model server network interface.
 ```json
```

Comment:
if this operation is required please change below part to:
```suggestion
```bash
echo '{
 "inputs": {
     "serving_default_inputs": "inputs"
     },
 "outputs":{
         "StatefulPartitionedCall:0": "outputs"
     }
}' > universal-sentence-encoder-multilingual/1/mapping_config.json
```
```